### PR TITLE
added jerkys to research

### PR DESCRIPTION
**Pull request by J3fftw1**
## Description

added jerkys to research + missing fertilizer and organic food
## Changes

added jerkys to research + missing fertilizer and organic food
dont know why it showed that i touched line 3
## Testability

- [x] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [x] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.

### CODE REVIEW

File: src/main/java/me/mrCookieSlime/Slimefun/Setup/ResearchSetup.java [hunk 1]
```diff
@@ -246,5 +247,10 @@ public static void setupResearches() {
 	    Slimefun.registerResearch(new Research(247, "GEO-Miner", 24), SlimefunItems.GEO_MINER);
 	    Slimefun.registerResearch(new Research(248, "Lightning Rune", 24), SlimefunItems.RUNE_LIGHTNING);
 	    Slimefun.registerResearch(new Research(249, "Totem of Undying", 36), SlimefunItems.TOTEM_OF_UNDYING);
+		Slimefun.registerResearch(new Research(250, "Pork Jerky", 2), SlimefunItems.PORK_JERKY);
+		Slimefun.registerResearch(new Research(251, "Chicken Jerky", 2), SlimefunItems.CHICKEN_JERKY);
+		Slimefun.registerResearch(new Research(252, "Mutton Jerky", 2), SlimefunItems.MUTTON_JERKY);
+		Slimefun.registerResearch(new Research(253, "Rabbit Jerky", 2), SlimefunItems.RABBIT_JERKY);
+		Slimefun.registerResearch(new Research(254, "Fish Jerky", 2), SlimefunItems.FISH_JERKY);
```
Comment (TheBusyBiscuit) [inline src/main/java/me/mrCookieSlime/Slimefun/Setup/ResearchSetup.java]: Bad indentation.
You could also argue that all the Jerky variants could belong to the same research.

**Comment by J3fftw1** [comment]
Its up to you what you prefer

**Comment by WalshyDev** [comment]
First off, reply to his comment don't comment again. Secondly, I agree with 1 research for all. It's stupid to make separate ones imo.

**Comment by SoSeDiK** [comment]
Not really related, but I've just noticed that there are two researches with ID 216 (for TRASH_CAN and CHARGING_BENCH). Can we quick fix that as well?

File: src/main/java/me/mrCookieSlime/Slimefun/Setup/ResearchSetup.java [hunk 2]
```diff
@@ -1,5 +1,6 @@
 package me.mrCookieSlime.Slimefun.Setup;
 
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SlimefunItem;
```
Comment (WalshyDev) [inline src/main/java/me/mrCookieSlime/Slimefun/Setup/ResearchSetup.java]: Unused import?

File: src/main/java/me/mrCookieSlime/Slimefun/Setup/ResearchSetup.java [hunk 3]
```diff
@@ -246,5 +246,6 @@ public static void setupResearches() {
 	    Slimefun.registerResearch(new Research(247, "GEO-Miner", 24), SlimefunItems.GEO_MINER);
 	    Slimefun.registerResearch(new Research(248, "Lightning Rune", 24), SlimefunItems.RUNE_LIGHTNING);
 	    Slimefun.registerResearch(new Research(249, "Totem of Undying", 36), SlimefunItems.TOTEM_OF_UNDYING);
+		Slimefun.registerResearch(new Research(250, "Charging Bench", 8), SlimefunItems.CHARGING_BENCH);
```
Comment (TheBusyBiscuit) [inline src/main/java/me/mrCookieSlime/Slimefun/Setup/ResearchSetup.java]: Also the indentation...

Comment (WalshyDev) [reply src/main/java/me/mrCookieSlime/Slimefun/Setup/ResearchSetup.java]: Stop using tabs REEEEEE

File: src/main/java/me/mrCookieSlime/Slimefun/Setup/ResearchSetup.java [hunk 4]
```diff
@@ -10,13 +10,13 @@
...
 	private ResearchSetup() {}
-	
+
 	public static void setupResearches() {
 	    Slimefun.registerResearch(new Research(0, "Walking Sticks", 1), SlimefunItems.GRANDMAS_WALKING_STICK, SlimefunItems.GRANDPAS_WALKING_STICK);
 	    Slimefun.registerResearch(new Research(1, "Portable Crafter", 1), SlimefunItems.PORTABLE_CRAFTER);
 	    Slimefun.registerResearch(new Research(2, "Fortune Cookie", 1), SlimefunItems.FORTUNE_COOKIE);
 	    Slimefun.registerResearch(new Research(4, "Portable Dustbin", 2), SlimefunItems.PORTABLE_DUSTBIN);
-	    Slimefun.registerResearch(new Research(5, "Beef Jerky", 2), SlimefunItems.BEEF_JERKY);
+	    Slimefun.registerResearch(new Research(5, "Beef Jerky", 2), SlimefunItems.BEEF_JERKY, SlimefunItems.FISH_JERKY, SlimefunItems.RABBIT_JERKY, SlimefunItems.MUTTON_JERKY, SlimefunItems.CHICKEN_JERKY, SlimefunItems.PORK_JERKY);
```
Comment (TheBusyBiscuit) [inline src/main/java/me/mrCookieSlime/Slimefun/Setup/ResearchSetup.java]: The research name should be changed accordingly too.

**Comment by WalshyDev** [comment]
You changed 1 tab but not the other.

File: src/main/java/me/mrCookieSlime/Slimefun/Setup/ResearchSetup.java [hunk 5]
```diff
@@ -16,7 +16,7 @@ public static void setupResearches() {
 	    Slimefun.registerResearch(new Research(1, "Portable Crafter", 1), SlimefunItems.PORTABLE_CRAFTER);
 	    Slimefun.registerResearch(new Research(2, "Fortune Cookie", 1), SlimefunItems.FORTUNE_COOKIE);
 	    Slimefun.registerResearch(new Research(4, "Portable Dustbin", 2), SlimefunItems.PORTABLE_DUSTBIN);
-	    Slimefun.registerResearch(new Research(5, "Beef Jerky", 2), SlimefunItems.BEEF_JERKY, SlimefunItems.FISH_JERKY, SlimefunItems.RABBIT_JERKY, SlimefunItems.MUTTON_JERKY, SlimefunItems.CHICKEN_JERKY, SlimefunItems.PORK_JERKY);
+	    Slimefun.registerResearch(new Research(5, "Jerky's", 2), SlimefunItems.BEEF_JERKY, SlimefunItems.FISH_JERKY, SlimefunItems.RABBIT_JERKY, SlimefunItems.MUTTON_JERKY, SlimefunItems.CHICKEN_JERKY, SlimefunItems.PORK_JERKY);
```
Comment (WalshyDev) [inline src/main/java/me/mrCookieSlime/Slimefun/Setup/ResearchSetup.java]: ...

Comment (TheBusyBiscuit) [inline src/main/java/me/mrCookieSlime/Slimefun/Setup/ResearchSetup.java]: Why the fuck the apostrophe???
There is no apostrophe...

Comment (J3fftw1) [reply src/main/java/me/mrCookieSlime/Slimefun/Setup/ResearchSetup.java]: i didnt know reeee changed rn sowwe cries in corner

File: src/main/java/me/mrCookieSlime/Slimefun/Setup/ResearchSetup.java [hunk 6]
```diff
@@ -246,5 +245,6 @@ public static void setupResearches() {
 	    Slimefun.registerResearch(new Research(247, "GEO-Miner", 24), SlimefunItems.GEO_MINER);
 	    Slimefun.registerResearch(new Research(248, "Lightning Rune", 24), SlimefunItems.RUNE_LIGHTNING);
 	    Slimefun.registerResearch(new Research(249, "Totem of Undying", 36), SlimefunItems.TOTEM_OF_UNDYING);
+        Slimefun.registerResearch(new Research(250, "Charging Bench", 8), SlimefunItems.CHARGING_BENCH);
```
Comment (TheBusyBiscuit) [inline src/main/java/me/mrCookieSlime/Slimefun/Setup/ResearchSetup.java]: The indentation, man...

**Pull request merged**